### ISSUE TITLE
Modifying country for demo

### ISSUE THREAD

**Issue by albfan** [kiegroup/optaweb-vehicle-routing#81]
Playing around with code, all works supernice! But trying to change the location (this time spain) we found a serie of points were project needs config:

Autocompletion for cities is hardcoded: I fixed with:

```
diff --git i/optaweb-vehicle-routing-frontend/src/ui/components/SearchBox.tsx w/optaweb-vehicle-routing-frontend/src/ui/components/SearchBox.tsx
index ef0c844..503971d 100644
--- i/optaweb-vehicle-routing-frontend/src/ui/components/SearchBox.tsx
+++ w/optaweb-vehicle-routing-frontend/src/ui/components/SearchBox.tsx
@@ -43,7 +43,7 @@ class SearchBox extends React.Component<IProps, IState> {
     searchDelay: 500,
   };
 
-  private searchProvider = new OpenStreetMapProvider({ params: { countrycodes: 'BE' } });
+  private searchProvider = new OpenStreetMapProvider({ params: { countrycodes: 'ES' } });
   private timeoutId: number;
 
   constructor(props: IProps) {
```

What would be the best way to setup this? maybe a property in npm or a property in backend accesible by REST API?

For started locations I modify to see something on Spain:

```
diff --git i/optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/service/demo/Belgium.java w/optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/service/demo/Belgium.java
index cc32776..6e02916 100644
--- i/optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/service/demo/Belgium.java
+++ w/optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/service/demo/Belgium.java
@@ -17,46 +17,8 @@
 package org.optaweb.vehiclerouting.service.demo;
 
 public enum Belgium {
-    AALST(50.933333, 4.033333),
-    ANDERLECHT(50.833333, 4.333333),
-    ANTWERP(51.217778, 4.400278),
-    BERINGEN(51.033333, 5.216667),
-    BRUGES(51.216667, 3.233333),
-    BRUSSELS(50.85, 4.35),
-    CHARLEROI(50.4, 4.433333),
-    CHATELET(50.4, 4.516667),
-    DENDERMONDE(51.033333, 4.1),
-    GEEL(51.166667, 5),
-    GENK(50.966667, 5.5),
-    GHENT(51.05, 3.733333),
-    HALLE(50.733333, 4.233333),
-    HASSELT(50.93, 5.3375),
-    IXELLES(50.833333, 4.366667),
-    KORTRIJK(50.833333, 3.266667),
-    LA_LOUVIERE(50.466667, 4.183333),
-    LEUVEN(50.883333, 4.7),
-    LIEGE(50.633333, 5.566667),
-    LIER(51.133333, 4.566667),
-    LOKEREN(51.1, 3.983333),
-    MECHELEN(51.016667, 4.466667),
-    MOLENBEEK(50.857778, 4.315833),
-    MONS(50.45, 3.95),
-    NAMUR(50.466667, 4.866667),
-    NINOVE(50.833333, 4.016667),
-    ROESELARE(50.933333, 3.116667),
-    SCHAERBEEK(50.866667, 4.383333),
-    SERAING(50.583333, 5.5),
-    SINT_NIKLAAS(51.166667, 4.133333),
-    SINT_TRUIDEN(50.8, 5.183333),
-    TIENEN(50.8, 4.933333),
-    TOURNAI(50.6, 3.383333),
-    TURNHOUT(51.316667, 4.95),
-    UCCLE(50.8, 4.333333),
-    VERVIERS(50.583333, 5.85),
-    VILVOORDE(50.933333, 4.416667),
-    WAREGEM(50.883333, 3.416667),
-    WAVRE(50.716667, 4.6),
-    YPRES(50.85, 2.883333);
+    GIJON(43.5314853,-5.7034741),
+    MADRID(40.4381311,-3.8196224);
 
     Belgium(double lat, double lng) {
         this.lat = lat;

```

Can we load that from a csv file or similar (Anything can work)

About start position, it seems to start in Belgium (but didn't find what does that)

![Captura de pantalla de 2019-03-21 16-46-56](https://user-images.githubusercontent.com/220968/54764969-fb346b80-4bf8-11e9-97e6-6d6744d040cf.png)

I can create a PR if you provide some guidance about how to approach this changes, in case they are interesting

**Comment by ge0ffrey** [comment]
Linking: https://github.com/kiegroup/optaweb-vehicle-routing/pull/95

**Comment by yurloc** [comment]
@albfan Solved by #97. Please evaluate and let me know if there's anything missing. It's now on master.
